pkg/controller/common/helpers.go

```diff
@@ -281,6 +284,50 @@ func ConvertRawExtIgnitionToV3(inRawExtIgn *runtime.RawExtension) (runtime.RawEx
 	return outRawExt, nil
 }
 
+// ConvertRawExtIgnitionToV3_4 ensures that the Ignition config in
+// the RawExtension is spec v3.4, or translates to it.
+func ConvertRawExtIgnitionToV3_4(inRawExtIgn *runtime.RawExtension) (runtime.RawExtension, error) {
+	// TODO(jkyros): since 3.4 is "ahead" of our current default 3.2, we're going "up" not down, which is
```

Comment:
Nit: I think this comment was for the V3_3 conversion (`Once we bump to 3.3 this will be briefly obsolete` should be 3.4, and the rest will be future proofing)

Reply:
You correct sir, good catch :smile: